$OpenBSD: patch-core_app_DigikamTarget_cmake,v 1.2 2019/06/16 15:14:56 rsadowski Exp $

Use generated Info.plist dependency. Fixes a problem in combination with Ninja
v1.9.0 because of duplicate rules for generating Info.plist.

Index: core/app/DigikamTarget.cmake
--- core/app/DigikamTarget.cmake.orig
+++ core/app/DigikamTarget.cmake
@@ -26,14 +26,15 @@ set(MACOSX_APP_DESCRIPTION             "Advanced digit
 set(MACOSX_BUNDLE_LONG_VERSION_STRING  ${DIGIKAM_VERSION_STRING})
 set(MACOSX_BUNDLE_SHORT_VERSION_STRING ${DIGIKAM_VERSION_SHORT})
 set(MACOSX_BUNDLE_BUNDLE_VERSION       ${DIGIKAM_VERSION_STRING})
-configure_file(${CMAKE_CURRENT_SOURCE_DIR}/../cmake/templates/Info.plist.cmake.in ${CMAKE_CURRENT_BINARY_DIR}/Info.plist)
 set_target_properties(digikam PROPERTIES MACOSX_BUNDLE_INFO_PLIST ${CMAKE_CURRENT_BINARY_DIR}/Info.plist)
+add_dependencies(digikam generate_info_header)
 
 target_link_libraries(digikam
                       PUBLIC
 
                       digikamcore
                       digikamgui
+                      digikamdatabase
 
                       Qt5::Core
                       Qt5::Gui
